dhall-openapi/openapi-to-dhall/Main.hs

```diff
@@ -60,6 +60,8 @@ import qualified Text.Megaparsec.Char.Lexer as Megaparsec.Lexer
 -- | Top-level program options
 data Options = Options
     { skipDuplicates :: Bool
+    , preferNaturalInt :: Bool
+    , natIntExceptions :: String
```

Comment:
```suggestion
    , natIntExceptions :: [(String, String)]
```

Would it be possible to do the string splitting within the command-line parsing logic so that the field can have this type?

Reply:
Done